apps/accounts/views/upstream_login.py

```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render
+from django.conf import settings
+import urllib.request as req
+
+
+def upstream_oauth_login(request):
+    mymedicare_login_url = 'https://dev2.account.mymedicare.gov/?scope=openid%20profile&client_id=bluebutton&state=54321'
```

Comment:
This URL should be a setting. Also, these parameters need to be added separately. For example, state needs to be dynamic, and client_id should possibly also be configurable